pyvista/plotting/plotting.py

```diff
@@ -5041,26 +5041,53 @@ def add_lines(self, lines, color='w', width=5, label=None, name=None):
             If an actor of this name already exists in the rendering window, it
             will be replaced by the new actor.
 
+        connected : bool, default: False
+            Treat ``lines`` as points representing a series of *connected* lines.
+            For example, two connected line segments would be represented as
+            ``np.array([[0, 0, 0], [1, 0, 0], [1, 1, 0]])``. If ``False``, an *even*
+            number of points must be passed to ``lines``, and the lines need not be
+            connected.
+
+
         Returns
         -------
         vtk.vtkActor
             Lines actor.
 
         Examples
         --------
+        Plot two lines.
+
         >>> import numpy as np
         >>> import pyvista
         >>> pl = pyvista.Plotter()
         >>> points = np.array([[0, 1, 0], [1, 0, 0], [1, 1, 0], [2, 0, 0]])
-        >>> actor = pl.add_lines(points, color='yellow', width=3)
+        >>> actor = pl.add_lines(points, color='purple', width=3)
+        >>> pl.camera_position = 'xy'
+        >>> pl.show()
+
+        Adding lines with ``connected=True`` will add a series of connected
+        lines segments.
+
+        >>> import numpy as np
+        >>> import pyvista
+        >>> pl = pyvista.Plotter()
+        >>> points = np.array([[0, 1, 0], [1, 0, 0], [1, 1, 0], [2, 0, 0]])
+        >>> actor = pl.add_lines(
+        ...     points, color='purple', width=3, connected=True
+        ... )
```

Comment:
We don't have to start from scratch here, I think we only have to create a new plotter. See e.g. [the docs of `collision()`](https://dev.pyvista.org/api/core/_autosummary/pyvista.PolyDataFilters.collision.html).

Reply:
that's neater - I'll fix that now, thanks